docs/spec/index.html

```diff
@@ -1,740 +1,918 @@
+        <h2>Presentations</h2>
         <p>
-          The above diagram illustrates how traceability-interop fits into the
-          larger family of W3C CCG standards.
-        </p>
+          Presentations in this spec are <a
+            href="https://w3c-ccg.github.io/traceability-vocab/#TraceablePresentation">Traceable
+            Presentations</a> which contain
+          a few notable identifiers that can be used for correlations,
+          retrievals, and business rule for processing of data.
+          <br />
+          <br />
+          These identifiers are:
         <ul>
-          <li>
-            At the <em>Business</em> layer, supply chain actors are exchanging
-            data in support of exchanged goods and services.
-          </li>
-          <li>
-            The <em>Trust</em> layer is supported by actors signing and
-            verifying data as Verifiable Credentials.
-          </li>
-          <li>
-            The <em>Process</em> defined standardized APIs for how to exchange
-            Verifiable Credentials (VC-API) and Verifiable Presentations
-            (trace-interop).
-          </li>
-          <li>
-            <em>Linked Data</em> establishes concise semantic context of the
-            data exchanged, enabled with JSON-LD by the trace-vocab
-            specification.
-          </li>
-          <li>
-            <em>Semantics</em> are defined in a multitude of established
-            vocabularies and come in many forms, from narrowly specialized,
-            industry-vertical vocabularies, to broadly-applicable, generic
-            vocabularies.
+          <li>`TraceablePresentation.id` the unique ID of the presentation
+            itself, think of this as identifying the wire transaction. This ID
+            MUST be a UUID v4 per [[rfc4122]].</li>
+          <li>`TraceablePresentation.workflow.definition` the ID for the
+            workflow definition, or type of workflow that the presented
+            credentials belong to. This ID MUST be a UUID v4 per [[rfc4122]].
           </li>
+          <li>`TraceablePresentation.workflow.instance` the ID for the unique
+            instance of the workflow - e.g. party A sending goods to party B
+            using workflow type X as defined by
+            `TraceablePresentation.workflow.definition`. This ID MUST be a UUID
+            v4 per [[rfc4122]].</li>
         </ul>
-      </section>
-
-      <section class="informative">
-        <h3>Workflows</h3>
+        </p>
         <p>
-          <em>Workflows</em> enable presenters to correlate individual presentations. A workflow <em>definition</em> can be used
-          to indicate that presentations relate to the same type of workflow.
+          As a rule, `TraceablePresentation.id` MUST be unique to each
+          presentation. A presentation received with a duplicate ID MUST be
+          rejected with a 403 Forbidden status code per [[rfc9110]]. A presenter
```

Comment:
`409 Conflict` seems much more appropriate than `403 Forbidden`... 
```suggestion
          rejected with a 409 Conflict status code per [[rfc9110]]. A presenter
```

Reply:
I like this much better